docs/dev/building-on-zksync/useful-address.md

```diff
@@ -16,12 +16,12 @@ Here are some contract addresses that may be useful and/or of interest to those
 | verifier_addr                 | [0x6A28B1bC6123fE525FF46d881Ff80871D0Eb3F02](https://etherscan.io/address/0x6A28B1bC6123fE525FF46d881Ff80871D0Eb3F02#code)|
 | mailbox_facet_addr            | [0x2EA0CFB9C942058ee5A84411EF2E37C6DE5bfe5c](https://etherscan.io/address/0x2EA0CFB9C942058ee5A84411EF2E37C6DE5bfe5c#code)|
 | l1_allow_list_addr            | [0x185e050EC1c22c45445a9AdEA99B12e1A1372eBA](https://etherscan.io/address/0x185e050EC1c22c45445a9AdEA99B12e1A1372eBA#code)|
-| l2_testnet_paymaster_addr     | [0x0265d9a5af8af5fe070933e5e549d8fef08e09f4](https://etherscan.io/address/0x0265d9a5af8af5fe070933e5e549d8fef08e09f4#code)|
+| l2_mainnet_paymaster_addr     | [0x0265d9a5af8af5fe070933e5e549d8fef08e09f4](https://goerli.explorer.zksync.io/address/0x0265d9a5af8af5fe070933e5e549d8fef08e09f4)|
```

Comment:
@krofax there is no mainnet paymaster, and especially not pointing to a goerli address. Just remove this line